t/rose-app-upgrade/05-cwd.t

```diff
@@ -38,12 +38,12 @@ CONFIG_DIR=$(cd ../config && pwd -P)
 run_pass "$TEST_KEY" rose app-upgrade --non-interactive \
  --meta-path=../rose-meta/ -C ../config fig
 file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<__OUTPUT__
-Current directory: $CONFIG_DIR
 [U] Upgrade_apple-fig: changes: 2
     namelist:add_sect_only=None=None
         Added
     =meta=test-app-upgrade/fig
         Upgraded from apple to fig
+Current directory: $CONFIG_DIR
```

Comment:
We'll need to go through tests with re-ordered results carefully, I'm not sure this one is harmless.

Reply:
I agree, I think that at least one reviewer needs to satisfy themselves of this.

Reply:
I've now been through the TB changes, I think this is the only suspect one.

Has the order of events actually changed or is it just the order the log messages are printed in that's changed?

Reply:
I think that I've now fixed this. It was being caused by wierd buffering problems. :*(